app/controllers/sign_out_controller.rb

```diff
@@ -0,0 +1,13 @@
+class SignOutController < ApplicationController
+  include FullyAuthenticatable
+
+  skip_before_action :handle_two_factor_authentication
+
+  def destroy
+    path_after_cancellation = decorated_session.cancel_link_path
+    sign_out
+    flash[:success] = t('devise.sessions.signed_out')
+    redirect_to path_after_cancellation
```

Comment:
No, it's  `decorated_session.cancel_link_path`, which is either `root_path` or `sign_up_start_path(request_id: sp_session[:request_id])` depending on whether the session is branded or not.